### PR TITLE
HPE CSI Driver for Kubernetes v.1.4.0

### PR DESCRIPTION
**Pull request by datamattsson**

### CODE REVIEW

**Comment by datamattsson** [comment]
```
~/code/helm3-charts/charts:mmattsson$ ct lint --charts hpe-csi-driver/v1.4.2
Linting charts...
Version increment checking disabled.

------------------------------------------------------------------------------------------------------------------------
 Charts to be processed:
------------------------------------------------------------------------------------------------------------------------
 hpe-csi-driver => (version: "1.4.2", path: "hpe-csi-driver/v1.4.2")
------------------------------------------------------------------------------------------------------------------------

Linting chart 'hpe-csi-driver => (version: "1.4.2", path: "hpe-csi-driver/v1.4.2")'
Validating /Users/mmattsson/code/helm3-charts/charts/hpe-csi-driver/v1.4.2/Chart.yaml...
Validation success! 👍
Validating maintainers...
==> Linting hpe-csi-driver/v1.4.2

1 chart(s) linted, 0 chart(s) failed
------------------------------------------------------------------------------------------------------------------------
 ✔︎ hpe-csi-driver => (version: "1.4.2", path: "hpe-csi-driver/v1.4.2")
------------------------------------------------------------------------------------------------------------------------
All charts linted successfully
```

File: charts/hpe-csi-driver/v1.4.2/questions.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  description: "Specify username for iSCSI CHAP authentication"
+  group: "HPE iSCSI settings"
+- variable: iscsi.chapPassword
+  label: "iSCSI CHAP Password"
+  type: password
+  required: false
+  description: "Specify password for iSCSI CHAP authentication"
+  group: "HPE iSCSI settings"
+- variable: registryPrefix
+  label: "registry"
```
Comment (pennyscissors) [inline charts/hpe-csi-driver/v1.4.2/questions.yml]: Nit: Please capitalize registry. This is what's displayed in the UI and is the only one in lowercase currently.

**Review by pennyscissors — COMMENTED** [review]
Left a nit comment, but it's pretty much ready to merge once that's settled.

File: charts/hpe-csi-driver/v1.4.2/questions.yml (hunk 1 above)
Comment (datamattsson) [reply charts/hpe-csi-driver/v1.4.2/questions.yml]: Thanks for catching this. There was another issue that had snuck in there. I'm correcting this in our upstream chart too.

**Pull request merged**